src/pseudopeople/configuration/interface.py

```diff
@@ -16,6 +16,10 @@
     $ user_config = {"decennial_census": {"row_noise": {"probability": {"omit_rows": 0.1},},},}
     $ psp.get_config("decennial_census", user_config)
 
+Note that when specifying a value to override in the configuration, users must specify the specific node they wish to
+change. Configuration is a hierarchical structure and to must properly source the lower levels. The configuration
```

Comment:
Typo: "and to must properly"

Reply:
I think I would change these three sentences to:

Note that the configuration is a hierarchical structure. When overriding default values, the entire path to the value of interest needs to be provided. The configuration path includes keys for dataset, column or row noise, noise type, and probability or other parameters unique to the specific noise type.

@rmudambi ?